.github/workflows/update-snapshots.yml

```diff
@@ -65,6 +65,9 @@ jobs:
       - name: Install
         run: pnpm i --no-frozen-lockfile
 
+      - name: Install Playwright and Firefox
```

Comment:
@anicyne, @sdvg warum brauchen wir das wirklich?

Reply:
Gleicher Commit wie bei diesem, bereits gemergten PR. Wurde hier nur cherry-picked: https://github.com/public-ui/kolibri/pull/7794

Hier der Build ohne diesen Commit: https://github.com/public-ui/kolibri/actions/runs/15302499792/job/43046513757
Wir sehen, dass die Browser nicht installiert waren: 
![screenshot 2025-05-30-09 39 11@2x](https://github.com/user-attachments/assets/e82500f6-606e-4100-be7b-af8f6ae72a2c)

Reply:
Hi, müssen wir die in allen packages nachinstallieren oder nur einmal in mono repo?

Wenn 2eres dann wprde ich den install auf root Ebene bevorzugen.